corehq/apps/domain/static/domain/js/info_basic.js

```diff
@@ -1,31 +1,72 @@
-hqDefine("domain/js/info_basic", [
-    'jquery',
-    'underscore',
-    'hqwebapp/js/select_2_ajax_widget', // for call center case owner
-    'select2/dist/js/select2.full.min',
-], function (
-    $,
-    _
-) {
-    $(function () {
-        $('#id_default_timezone').select2({
-            placeholder: gettext('Select a Timezone...'),
-        });
 
-        $('#id_call_center_enabled').change(function () {
-            var type = $('#id_call_center_type').closest('.control-group');
-            var case_owner = $('#id_call_center_case_owner').closest('.control-group');
-            var case_type = $('#id_call_center_case_type').closest('.control-group');
-            if ($(this).is(':checked')) {
-                type.removeClass('hide');
-                case_owner.removeClass('hide');
-                case_type.removeClass('hide');
-            } else {
-                type.addClass('hide');
-                case_owner.addClass('hide');
-                case_type.addClass('hide');
+hqDefine(
+    "domain/js/info_basic",
+    [
+        "jquery",
+        "underscore",
+        "@mapbox/mapbox-gl-geocoder/dist/mapbox-gl-geocoder.min",
+        "hqwebapp/js/initial_page_data",
+        "hqwebapp/js/select_2_ajax_widget", // for call center case owner
+        "select2/dist/js/select2.full.min",
+    ],
+    function ($, _, MapboxGeocoder,initialPageData) {
```

Comment:
I'll need to look at this more because I'm not familiar with this part of the codebase. But can you take screenshots and/or link to it deployed in staging so I can see what it's supposed to be like?

Reply:
![Screenshot from 2020-08-26 16-41-44](https://user-images.githubusercontent.com/8924132/91298076-fea02f00-e7bc-11ea-8c67-49185452b835.png)